framework/include/problems/DumpObjectsProblem.h

```diff
@@ -76,6 +76,12 @@ class DumpObjectsProblem : public FEProblemBase
   void addMaterial(const std::string & type,
                    const std::string & name,
                    InputParameters & parameters) override;
+  void addPostprocessor(std::string type,
```

Comment:
minor nitpick: You might make these const references too like the other declarations.

Reply:
Yeah, I agree, this was an oversight.